cmd/project_list.go

```diff
@@ -44,13 +46,14 @@ var projectListCmd = &cobra.Command{
 			ListOptions: gitlab.ListOptions{
 				PerPage: num,
 			},
-			Simple:     gitlab.Bool(true),
-			OrderBy:    gitlab.String("id"),
-			Sort:       gitlab.String("asc"),
-			Owned:      gitlab.Bool(projectListConfig.Owned),
-			Membership: gitlab.Bool(projectListConfig.Membership),
-			Starred:    gitlab.Bool(projectListConfig.Starred),
-			Search:     gitlab.String(search),
+			Simple:           gitlab.Bool(true),
+			OrderBy:          gitlab.String("id"),
+			Sort:             gitlab.String("asc"),
+			Owned:            gitlab.Bool(projectListConfig.Owned),
+			Membership:       gitlab.Bool(projectListConfig.Membership),
+			Starred:          gitlab.Bool(projectListConfig.Starred),
+			SearchNamespaces: gitlab.Bool(projectListConfig.Group),
```

Comment:
I simply pass the flag `-g` as boolean parameter wich sets the `SearchNamespaces` boolean for the gitlab SDK github.com/xanzy/go-gitlab [SDK](https://pkg.go.dev/github.com/xanzy/go-gitlab#ListProjectsOptions)

which translates into gitlab [API](https://docs.gitlab.com/ee/api/projects.html#list-all-projects) call.

Reply:
Is it clear?